### PR TITLE
Adding method for triggering an event from the platform.

### PR DESCRIPTION
**Pull request by CTDoriott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 <!--
 
-TODO: Uncomment when php sdk published to public packagist:
+TODO: Uncomment when php sdk is published to public packagist:
```
Comment (crfroehlich) [inline README.md]: This is published now, no? @davehensleyns8

File: public/en/platform/protect-sdk-php/action-documentation.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ The purpose of the Action Client is to enable calls to NS8 vi the API for managi
   * The ability to pass new, unknown (to the SDK) events to the API for processing
   * Interaction with the NS8 API without specifically utilizing HTTP components or needing to customize request logic
 
+### Actions VS Events
```
Comment (crfroehlich) [inline public/en/platform/protect-sdk-php/action-documentation.md]: Might be worth linking to the overall docs:

* https://github.com/ns8inc/protect-integration-docs/blob/dev/public/en/platform/protect-integration-docs/switchboards_foundational_switches.md
* https://github.com/ns8inc/protect-integration-docs/blob/dev/public/en/platform/protect-integration-docs/switchboards_actions_v_events.md

**Comment by crfroehlich** [comment]
Also minor quibble, the PR title should be `Adding method for triggering an event from the platform.`

File: README.md (hunk 1 above)
Comment (CTDoriott) [reply README.md]: @crfroehlich I've restored the original README.md from Dev though to stay on the side of caution.

**Comment by CTDoriott** [comment]
@crfroehlich Changes requested should now be addressed. Thanks for updating the PR title as well!

**Pull request merged**